nix/azure.nix

```diff
@@ -389,9 +389,9 @@ in
 
     };
 
-    fileSystems = mkOption {
-      type = with types; loaOf (submodule fileSystemsOptions);
-    };
+    #fileSystems = mkOption {
+    #  type = with types; loaOf (submodule fileSystemsOptions);
```

Comment:
`loaOf` is an alias to `attrsOf` now, so why not just change the type to `attrsOf` proper and be done with it? Doesn't seem necessary to comment it out and break things.